sites/public/jest.config.js

```diff
@@ -5,7 +5,16 @@ process.env.TZ = "UTC"
 
 module.exports = {
   testRegex: ["/*.test.tsx$", "/*.test.ts$"],
-  collectCoverageFrom: ["**/*.ts", "!**/*.tsx"],
+  collectCoverageFrom: ["**/*.ts", "**/*.tsx"],
+  coveragePathIgnorePatterns: [
+    "cypress",
+    "types",
+    "__tests__",
+    "next-env.d.ts",
+    "sentry.client.config.ts",
+    "sentry.server.config.ts",
+    "sentry.edge.config.ts",
+  ],
```

Comment:
Should this be added to partners and shared-helpers?

Reply:
These files are specific to the public site, and looks like code cov is already setup on partners/shared-helpers